test/unit/app/controllers/swaps-test.js

```diff
@@ -121,12 +121,14 @@ const EMPTY_INIT_STATE = {
     topAggId: null,
     routeState: '',
     swapsFeatureIsLive: false,
+    swapsQuoteRefreshTime: 0,
   },
 }
 
 const sandbox = sinon.createSandbox()
 const fetchTradesInfoStub = sandbox.stub()
 const fetchSwapsFeatureLivenessStub = sandbox.stub()
+const fetchSwapsQuoteRefreshTimeStub = sandbox.stub()
```

Comment:
Note that this stub isn't being reset correctly between tests, but we can solve that separately in a later PR. Might as well fix all of these stubs at once, since they all follow the same pattern.